packages/split-button/src/SplitButton/SplitButton.types.ts

```diff
@@ -0,0 +1,111 @@
+import { ComponentType, ReactFragment } from 'react';
+
+import {
+  type ButtonProps as ImportedButtonProps,
+  Variant as ButtonVariants,
+} from '@leafygreen-ui/button';
+import { DarkModeProps } from '@leafygreen-ui/lib';
+import { MenuItemProps } from '@leafygreen-ui/menu';
+import { type MenuProps as ImportedMenuProps } from '@leafygreen-ui/menu';
+import {
+  Align as ImportedAlign,
+  Justify as ImportedJustify,
+} from '@leafygreen-ui/popover';
+
+export const Variant = {
+  Default: ButtonVariants.Default,
+  Primary: ButtonVariants.Primary,
+  Danger: ButtonVariants.Danger,
+} as const;
+
+export type Variant = typeof Variant[keyof typeof Variant];
+
+type ButtonProps = Omit<
+  ImportedButtonProps,
+  'rightGlyph' | 'href' | 'as' | 'variant'
+>;
+
+export type MenuItemsType = ReactFragment & {
+  props: {
+    children: Array<ComponentType<MenuItemProps>>;
+  };
+};
+
+export const Align = {
+  Top: ImportedAlign.Top,
+  Bottom: ImportedAlign.Bottom,
+};
+
+export type Align = typeof Align[keyof typeof Align];
+
+export const Justify = {
+  Start: ImportedJustify.Start,
+  End: ImportedJustify.End,
+};
+
+export type Justify = typeof Justify[keyof typeof Justify];
+
+export type Test = typeof Justify;
+
+export type OmittedMenuProps = Omit<
+  ImportedMenuProps,
+  | 'children'
+  | 'trigger'
+  | 'open'
+  | 'setOpen'
+  | 'shouldClose'
+  | 'darkMode'
+  | 'onClick'
+  | 'align'
+  | 'justify'
+  | 'className'
+  | 'refEl'
+>;
+
+export interface MenuProps extends OmittedMenuProps {
```

Comment:
this reads weirdly MenuProps extends OmittedMenuProps should probably rename to SelectedMenuProps (or something similar)